packages/wallet/contract/src/smartWallet.js

```diff
@@ -19,7 +19,7 @@ const { assign, entries, keys, fromEntries } = Object;
  * agoricNames: NameHub,
  * board: Board
  * namesByAddress: NameHub,
- * storageNode?: ERef<StorageNode>,
+ * storageNode: ERef<StorageNode>,
```

Comment:
remind me how this works in the sim chain? I don't see how it can work if the root `chainStorage` is resolved to `null`:

https://github.com/Agoric/agoric-sdk/blob/16ef8776e28ea334fe037c928b531e4be5b54ab8/packages/vats/src/core/chain-behaviors.js#L296-L305

Reply:
maybe it just doesn't?

Reply:
Yeah, `chainStorage` resolves to null . Then this helper accommodates that: https://github.com/Agoric/agoric-sdk/blob/15720474802c5630210184602bcc29811c0a0a21/packages/vats/src/lib-chainStorage.js#L85-L97

Reply:
very well.